sound/soc/sof/intel/hda-sdw-bpt.c

```diff
@@ -115,6 +239,18 @@ static int hda_sdw_bpt_dma_disable(struct device *dev,
 	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
 	int ret;
 
+	if (!sdev->dspless_mode_selected) {
+		snd_hdac_ext_stream_clear(sdw_bpt_stream);
+
+		ret = chain_dma_trigger(sdev,
+					hdac_stream(sdw_bpt_stream)->stream_tag,
+					hdac_stream(sdw_bpt_stream)->direction,
+					SOF_IPC4_PIPE_PAUSED);
+		if (ret < 0)
+			dev_err(sdev->dev, "%s: chain_dma_trigger PIPE_PAUSED failed: %d\n",
+				__func__, ret);
+	}
+
```

Comment:
Should this patch to be moved earlier? can BRA work for rt711 before this patch?

Reply:
> Should this patch to be moved earlier? can BRA work for rt711 before this patch?

Yes, rt711 can work without this in dsp less mode. But, yes, it is better to move the commit before the codec change.

Reply:
it is unlikely that anyone would use DSPless mode apart from developers for testing.